metadata.yaml

```diff
@@ -109,22 +248,45 @@ runtimes:
     name: 10.x (RHOAR)
 - id: spring-boot
   name: Spring Boot
-  description: Create stand-alone, production-grade Spring based Applications that you can "just run".
-  icon: data:image/svg+xml;charset=utf8,%3Csvg xmlns='http://www.w3.org/2000/svg' width='205.3' height='184.28'%3E%3Cpath fill='%2368bd45' d='M202.66 81.08l-40.43-70C158.72 5 150.1 0 143.08 0H62.22c-7 0-15.64 5-19.15 11.06l-40.43 70c-3.52 6.08-3.52 16 0 22.12l40.43 70c3.51 6.08 12.13 11.06 19.15 11.06h80.86c7 0 15.63-5 19.15-11.06l40.43-70c3.52-6.07 3.52-16.01 0-22.1zM93.91 38.72a8.34 8.34 0 1 1 16.67 0v49.81a8.34 8.34 0 1 1-16.67 0zM102.25 145A56.51 56.51 0 0 1 68.54 43.17a7.41 7.41 0 1 1 8.85 11.89 41.69 41.69 0 1 0 66.54 33.47A41.89 41.89 0 0 0 127 55a7.41 7.41 0 0 1 8.76-12 56.52 56.52 0 0 1-33.51 102z'/%3E%3C/svg%3E
+  description: Create stand-alone, production-grade Spring based Applications that
+    you can "just run".
+  icon: data:image/svg+xml;charset=utf8,%3Csvg xmlns='http://www.w3.org/2000/svg'
+    width='205.3' height='184.28'%3E%3Cpath fill='%2368bd45' d='M202.66 81.08l-40.43-70C158.72
+    5 150.1 0 143.08 0H62.22c-7 0-15.64 5-19.15 11.06l-40.43 70c-3.52 6.08-3.52 16
+    0 22.12l40.43 70c3.51 6.08 12.13 11.06 19.15 11.06h80.86c7 0 15.63-5 19.15-11.06l40.43-70c3.52-6.07
+    3.52-16.01 0-22.1zM93.91 38.72a8.34 8.34 0 1 1 16.67 0v49.81a8.34 8.34 0 1 1-16.67
+    0zM102.25 145A56.51 56.51 0 0 1 68.54 43.17a7.41 7.41 0 1 1 8.85 11.89 41.69 41.69
+    0 1 0 66.54 33.47A41.89 41.89 0 0 0 127 55a7.41 7.41 0 0 1 8.76-12 56.52 56.52
+    0 0 1-33.51 102z'/%3E%3C/svg%3E
   metadata:
     pipelinePlatform: maven
   versions:
   - id: current-community
-    name: 1.5.13.RELEASE (Community)
+    name: 1.5.14.RELEASE ( Community )
   - id: current-redhat
-    name: 1.5.13.RELEASE (RHOAR)
+    name: 1.5.14.RELEASE ( RHOAR )
   - id: current-osio
-    name: 1.5.12.RELEASE (OSIO)
+    name: 1.5.14.RELEASE ( OSIO )
```

Comment:
These spaces look weird but I guess this is done automatically and is not configurable?

Reply:
Actually, this is a mistake on my part that I have fixed after that PR :)